README.md

```diff
@@ -83,35 +86,50 @@ downsample([x], y, n_out, **kwargs) -> ndarray[uint64]
+
 ### Downsampling algorithms 📈
 
 The following downsampling algorithms (classes) are implemented:
 
-| Downsampler | Description | `**kwargs` |
-| ---:| --- |--- |
-| `MinMaxDownsampler` | selects the **min and max** value in each bin | `n_threads` |
-| `M4Downsampler` | selects the [**min, max, first and last**](https://dl.acm.org/doi/pdf/10.14778/2732951.2732953) value in each bin | `n_threads` |
-| `LTTBDownsampler` | performs the [**Largest Triangle Three Buckets**](https://skemman.is/bitstream/1946/15343/3/SS_MSthesis.pdf) algorithm | `n_threads` |
-| `MinMaxLTTBDownsampler` | (*new two-step algorithm 🎉*) first selects `n_out` * `minmax_ratio` **min and max** values, then further reduces these to `n_out` values using the **Largest Triangle Three Buckets** algorithm | `n_threads`, `minmax_ratio`<sup>*</sup> |
+|             Downsampler | Description                                                                                                                                                                                       | `**kwargs`                               |
+| ----------------------: | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------- |
+|     `MinMaxDownsampler` | selects the **min and max** value in each bin                                                                                                                                                     | `n_threads`                              |
+|         `M4Downsampler` | selects the [**min, max, first and last**](https://dl.acm.org/doi/pdf/10.14778/2732951.2732953) value in each bin                                                                                 | `n_threads`                              |
+|       `LTTBDownsampler` | performs the [**Largest Triangle Three Buckets**](https://skemman.is/bitstream/1946/15343/3/SS_MSthesis.pdf) algorithm                                                                            | `n_threads`                              |
+| `MinMaxLTTBDownsampler` | (_new two-step algorithm 🎉_) first selects `n_out` \* `minmax_ratio` **min and max** values, then further reduces these to `n_out` values using the **Largest Triangle Three Buckets** algorithm | `n_threads`, `minmax_ratio`<sup>\*</sup> |
 
-<sup>*</sup><i>Default value for `minmax_ratio` is 30, which is empirically proven to be a good default. (More details in our upcomming paper)</i>
+<sup>\*</sup><i>Default value for `minmax_ratio` is 30, which is empirically proven to be a good default. (More details in our upcoming paper)</i>
 
+### Handling NaNs
+
+This library supports two `NaN`-policies:
+
+1. Omit `NaN`s (`NaN`s are ignored during downsampling).
+2. Return `NaN` once there is one present in the bin of the considered data.
```

Comment:
What happens when multiple `NaN`s are present? Which one's index will be returned?

Reply:
Index of first `NaN` will be returned, I'll update the README